content/en/docs/languages/ruby/_index.md

```diff
@@ -1,12 +1,16 @@
 ---
 title: Ruby
-api_path: https://rubydoc.info/gems/grpc
-no_list: true
+api_path: grpc/LANG/namespace_grpc
```

Comment:
This powers the redirect for /docs/languages/ruby/api , but it seems wrong. On the preview server it redirects inappropriately: https://deploy-preview-970--grpc-io.netlify.app/docs/languages/ruby/api . Seems this should remain rubydoc.info.

Reply:
Thx for your review! Sorry about the missing check.
fixed it on: https://github.com/grpc/grpc.io/pull/970/commits/3c4f8f0538fc0068bad446cb8176087cdca77b12